transformers/test_data/vow_heal.go

```diff
@@ -0,0 +1,55 @@
+package test_data
+
+import (
+	"math/rand"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+)
+
+var rawVowHealLog = types.Log{
+	Address: common.HexToAddress(VatAddress()),
```

Comment:
Should this be - `common.HexToAddress(VowAddress())`?

Reply:
Yes - nice catch!